core/arch/arm/plat-vexpress/main.c

```diff
@@ -194,15 +194,35 @@ service_init(init_tzc400);
 #endif /*CFG_TZC400*/
 
 #if defined(PLATFORM_FLAVOR_qemu_virt)
+static void release_secondary_early_hpen(size_t pos)
+{
+	struct mailbox {
+		uint64_t ep;
+		uint64_t hpen[];
+	} *mailbox;
+
+	assert(pos < CFG_TEE_CORE_NB_CORE);
```

Comment:
You've just checked that pos is OK in `psci_cpu_on()` (but there checking against pos == 0 too).

Reply:
true. i will remove it.